docs/usage/examples/opentelemetry.md

```diff
@@ -0,0 +1,115 @@
+# OpenTelemetry
+
+Requirements:
+
+- docker-compose
+
+## Prepare setup
+
+Create a `docker-compose.yaml` and `otel-collector-config.yml` file as seen below in a folder
```

Comment:
Several lines in this files, like this one, do not end with a full stop. Can you review all sentences in this file to correct that?

Reply:
fixed